src/resources/help/en_US/relnotes3.4.0.html

```diff
@@ -54,7 +54,8 @@
 <a name="File_format"></a>
 <p><b><font size="4">File format reader changes in version 3.4</font></b></p>
 <ul>
-    <li>feature1</li>
+    <li>HDF 4 based readers were removed: Cosmos, RAGE, ZeusMP</li>
+    <li>HDF 4 support in the Enzo reader was removed</li>
```

Comment:
Is it worth adding an item to `Build_changes` that HDF4 support was removed from VisIt?

Reply:
There is no 'Build_changes', there is 'Build_features' and the description is for features **_added_**.
There is also 'Configuration_changes', and 'Changes_for_VisIt_developers', which section would you suggest?

Reply:
I would put it under "Build_features" and change the description in the section from "Build features added in version 3.4" to "Changes to build_visit in version 3.4". That would make it consistent with the description in the sections list at the top of the release notes. I would also change the description in the sections list from "Changes for build_visit" to "Changes to build_visit".

I also noticed that the sections list has "General_Features", "Advanced_Features" and "GUI_Changes". The second word in all those names should start with a lower case letter to be consistent with the rest of the names and to match the names further down.

Reply:
Updated.